node_registry.schema.json

```diff
@@ -1,100 +1,136 @@
 {
-	"$schema": "https://json-schema.org/draft/2020-12/schema",
-    "patternProperties": {
-    	"^[a-zA-Z0-9_-]+$": {
-        	"type": "object",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "patternProperties": {
+    "^[a-zA-Z0-9_-]+$": {
+      "type": "object",
+      "properties": {
+        "url": {
+          "type": "string",
+          "format": "uri",
+          "pattern": "^https?://"
+        },
+        "date_added": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "affiliation": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "icon_url": {
+          "type": "string",
+          "format": "uri",
+          "pattern": "^https?://"
+        },
```

Comment:
See `icon` relationship name.

Reply:
This isn't part of the `links` list since it refers to an optional icon for the whole node, not a single service.

Also, this value is set manually when registering the node so it's easier for the user to add/update in this format.

Reply:
To consider in a later PR